spi-fly/spi-fly-static-bundle/pom.xml

```diff
@@ -78,6 +78,24 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>properties-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>read-project-properties</goal>
+                        </goals>
+                        <configuration>
+                            <files>
+                                <file>../spi-fly-core/src/main/resources/org/apache/aries/spifly/packageinfo</file>
```

Comment:
I tried using the `${parent.basedir}` placeholder here to make the lookup dynamic but it seems placeholders are not supported here.